clearpath_sensors/launch/velodyne_lidar.launch.py

```diff
@@ -68,7 +68,7 @@ def generate_launch_description():
         output='screen',
         parameters=[parameters],
         remappings=[
-          ('/diagnostics', 'diagnostics')
+          ('/diagnostics', 'diagnostics'), ('/tf', 'tf'), ('/tf_static', 'tf_static')
```

Comment:
Can you please put each remap on a new line to be consistent with our other launch files